.github/workflows/release-pkg.nu

```diff
@@ -107,31 +112,34 @@ if $os in ['windows-latest'] {
 let suffix = if $os == 'windows-latest' { '.exe' }
 # nu, nu_plugin_* were all included
 let executable = $'target/($target)/release/($bin)*($suffix)'
-$'Current executable file: ($executable)'
+print $'Current executable file: ($executable)'
 
 cd $src; mkdir $dist;
 rm -rf $'target/($target)/release/*.d' $'target/($target)/release/nu_pretty_hex*'
-$'(char nl)All executable files:'; hr-line
-ls -f $executable
+print $'(char nl)All executable files:'; hr-line
+# We have to use `print` here to make sure the command output is displayed
```

Comment:
do we need to specify that here?

i'm wondering if it should be added to all the other `print`s then :open_mouth:

Reply:
I thought `ls -f $executable` will display its output automatically just as `[LICENSE $executable] | each {|it| cp -rv $it $dist } | flatten` does( I did not use `print` here), but it didn't, that's why I add `print` here.
Other `print`s just work as expected since nu v0.78

Reply:
oooh you are right here :+1:

`ls` won't print anything unless commanded to :smirk:

Reply:
ok `cp` does print things indeed with the `-v` option :+1: 

i'd say if goes probably to `stderr` and thus does not need that `print` :thinking:

Reply:
maybe not standard err, but the behaviour looks the same as `print` :relieved: